repos/iiif-storage/src/Controller/CollectionController.php

```diff
@@ -44,6 +44,9 @@ public function __construct(CollectionRepository $repo, CollectionBuilder $build
         $this->settingsHelper = $settingsHelper;
     }
 
+    // @todo move this configuration.
+    const PER_PAGE = 24;
```

Comment:
Added 24 for its divisibility. Not sure if we do want this to be completely configurable, maybe a multiple choice. The settings is available in this class so can be added in quickly.